src/requests/team.ts

```diff
@@ -113,7 +113,7 @@ export async function createTeam(
     slug,
     theme: {
       logo,
-      primary: primaryColor,
+      primary_colour: primaryColor,
```

Comment:
nit: `primary_colour` & `primaryColor` feel easily to confuse here - can we consistently use one or the other?